hera_opm/data/idr2/task_scripts/do_LSTBIN.sh

```diff
@@ -24,10 +27,13 @@ min_N=${3}
 rephase=${4}
 ntimes_per_file=${5}
 lst_start=${6}
-vis_units=${7}
-output_file_select=${8}
+dlst=${7}
+vis_units=${8}
+output_file_select=${9}
+file_ext=${10}
+outdir=${11}
 data_files=($@)
-data_files=(${data_files[*]:8})
+data_files=(${data_files[*]:11})
```

Comment:
This number seems a little brittle, so you may want to say in a comment what part of the file name you're selecting and what structure you're expecting it to have.

Reply:
ah, I'm not selecting part of the filename. its selecting all of the kwargs past kwarg 11 (which is `outdir`)

Reply:
Ah okay, I misunderstood.